### ISSUE TITLE
xml import: whitespace handling

### ISSUE THREAD

**Issue by sarmbruster** [THM-Graphs/neo4j-atag#7]
https://gitlab.rlp.net/adwmainz/digicademy/sbw/tei2json/-/blob/master/xsl/standoff_property.xsl#L25-46

Spec: https://wiki.tei-c.org/index.php/XML_Whitespace

**Comment by sarmbruster** [comment]
proposed signature: 

```
CALL atag.chain.update(
<uuidText>,
<uuidCharacterBefore>, // evtl. null
<uuidCharacterAfter>, // evtl. null
[
{uuid: <characterUUID>, character: <character>},
{uuid: <characterUUID>, character: <character>},
{uuid: <characterUUID>, character: <character>},
{uuid: <characterUUID>, character: <character>},
]
) YIELD charactersNew, charactersDeleted, charactersExisting
```

**Comment by sarmbruster** [comment]
fixed in v0.2.6
